test/cypress/integration/group-proposals.spec.js

```diff
@@ -1,4 +1,5 @@
-const userId = Math.floor(Math.random() * 10000)
+const userId = 100
+// const userId = Math.floor(Math.random() * 10000)
```

Comment:
Is this change supposed to be here? cc @sandrina-p

Reply:
no